tcms/rpc/api/testexecution.py

```diff
@@ -183,12 +183,27 @@ def update(execution_id, values, **kwargs):
 
     # if this call updated TE.status then adjust timestamps
     if values.get("status"):
+        now = timezone.now()
         if test_execution.status.weight != 0:
-            test_execution.close_date = timezone.now()
+            test_execution.close_date = now
```

Comment:
It's fine if timestamps on TE and TR are slightly different by a few milliseconds but that's up to you if you want to change it back.